qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/commons/snippets/SnippetRegistry.java

```diff
@@ -235,11 +235,16 @@ private static Snippet createSnippet(JsonReader reader,
 	 * @param canSupportMarkdown true if markdown is supported to generate
 	 *                           documentation and false otherwise.
 	 * @param contextFilter      the context filter.
+	 * @param suffixToFind      suffix to found to eat it when completion snippet

```

Comment:
suffix to find, that will be removed when completion snippet is applied

Reply:
fixed